benchmark/jsonresultset/jsonresultset_test.go

```diff
@@ -49,15 +49,16 @@ func runJSONResultSet() {
 		case <-ctx.Done():
 		}
 	}()
-	setCustomConfig()
-	cfg, err := sf.GetConfigFromEnv([]sf.ConfigParam{
-		{"Account", "SNOWFLAKE_TEST_ACCOUNT", true},
-		{"User", "SNOWFLAKE_TEST_USER", true},
-		{"Password", "SNOWFLAKE_TEST_PASSWORD", true},
-		{"Role", "SNOWFLAKE_TEST_ROLE", false},
-		{"Host", "SNOWFLAKE_TEST_HOST", false},
-		{"Port", "SNOWFLAKE_TEST_PORT", false},
-		{"Protocol", "SNOWFLAKE_TEST_PROTOCOL", false},
+	setCustomJSONDecoder()
+	setMaxChunkDownloadWorkers()
+	cfg, err := sf.GetConfigFromEnv([]*sf.ConfigParam{
+		{Name: "Account", EnvString: "SNOWFLAKE_TEST_ACCOUNT", FailOnMissing: true},
```

Comment:
`EnvString` is a strange name. Maybe just `env`?

Reply:
Replaced it with `EnvName` as it seems to be a standard in naming the environment variables in Go SDK. I have found 10 occurrences of `*EnvName` in the Go SDK.